### DISCUSSION TITLE
Ionic React Capacitor google maps

### DISCUSSION THREAD

**Discussion by VinceSanityyy** [ionic-team/capacitor#3387] [Q&A] [answered]
Hello ionic team, I am developing an ionic app using the react framework with google maps functions it, is there any plugin for google maps using react/capacitor since react only uses capacitor? I can only find cordova's google maps plugin and i tried `react-google-maps` and its not working.  Please help. Thanks

**Comment by imhoffd** [comment, marked as answer]
You can follow this proposal for a Google Maps plugin in Capacitor: https://github.com/capacitor-community/proposals/issues/20

**Reply by VinceSanityyy** [reply]
okay thanks
